caddy/setup/markdown.go

```diff
@@ -55,9 +34,9 @@ func markdownParse(c *Controller) ([]*markdown.Config, error) {
 
 	for c.Next() {
 		md := &markdown.Config{
-			Renderer:    blackfriday.HtmlRenderer(0, "", ""),
-			Templates:   make(map[string]string),
-			StaticFiles: make(map[string]string),
+			Renderer:   blackfriday.HtmlRenderer(0, "", ""),
+			Extensions: make(map[string]struct{}),
```

Comment:
```go
Extensions: make(map[string]bool),
```

I don't see why we use a ```struct{}``` here; a bool will do.

(```map[string]RendererInterface``` would be an extension point, btw.)